File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/DebugUtil.java
package org.mifos.mobilewallet.mifospay.utils;

import android.util.Log;

/**
 * Created by ankur on 26/June/2018
 */

public class DebugUtil {

    public static Object[] log(Object... objects) {

        StringBuilder stringBuilderToPrint = new StringBuilder();
        for (Object object : objects) {
            stringBuilderToPrint.append(object).append(", ");
        }
        String stringToPrint = stringBuilderToPrint.toString();
        stringToPrint = stringToPrint.substring(0, stringToPrint.lastIndexOf(','));
        Log.d("QXZ:: ", stringToPrint);
        return objects;
    }
}

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/DebugUtil.kt
package org.mifos.mobilewallet.mifospay.utils

import android.util.Log

/**
 * Created by ankur on 26/June/2018
 */
object DebugUtil {
    @JvmStatic
    fun log(vararg objects: Any): Array<out Any> {
        var stringToPrint = ""
        for (`object` in objects) {
            stringToPrint += "$`object`, "
        }
        stringToPrint = stringToPrint.substring(0, stringToPrint.lastIndexOf(','))
        Log.d("QXZ:: ", stringToPrint)
        return objects
    }
}
